docs/ThemeColors.stories.tsx

```diff
@@ -32,7 +58,12 @@ LightThemeColors.args = {
 export const DarkThemeColors = Template.bind({});
 
 DarkThemeColors.args = {
-  swatchData: tokens.dark.colors,
+  swatchData: Object.fromEntries(
+    Object.entries(tokens.dark.colors).filter(
+      (key) => key.indexOf('networks') < 0,
+    ),
+  ),
+  networkData: tokens.dark.colors.networks,
   borderColor: tokens.dark.colors.border.muted.value,
   textBackgroundColor: tokens.dark.colors.background.default.value,
   textColor: tokens.dark.colors.text.default.value,
```

Comment:
Had to do this because the doc components only handle one level of nesting. This could probably be refactored to handle as much nesting as possible but will create a new issue for that